step-templates/file-system-create-file.json

```diff
@@ -1,52 +1,59 @@
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.RunOnServer": "false",
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
   },
-  "SensitiveProperties": {},
   "Parameters": [
     {
-      "Name": "FilePath",
-      "Label": "The full path where the file will be created.",
-      "HelpText": "Provide the entire physical path to which the physical file will be created. For example, 'C:\\Temp\\MyFile.txt' will create am empty text file named 'MyFile.txt' in the 'Temp' folder on the C: drive.",
-      "DefaultValue": null,
+      "Id": "607c3a21-a0ab-4b0d-aa01-0864bf2ae909",
+      "Name": "FileName",
+      "Label": "File name with extension to create on the file system.",
+      "HelpText": "Valid file name to be created under given path.",
+      "DefaultValue": "",
       "DisplaySettings": {
         "Octopus.ControlType": "SingleLineText"
-      }
+      },
+      "Links": {}
     },
     {
-      "Name": "FileContent",
-      "Label": "File Content",
-      "HelpText": "The text you would like inside the file, if nothing then will create an empty file.",
-      "DefaultValue": null,
+      "Id": "0e342c97-2613-4e13-9c1e-193666eff1c0",
+      "Name": "Path",
+      "Label": "Folder path under which specified file to be created.",
+      "HelpText": "Specify folder path, where you wish to create specified file.",
+      "DefaultValue": "",
       "DisplaySettings": {
-        "Octopus.ControlType": "MultiLineText"
-      }
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
     },
     {
-      "Id": "43aff0e0-e7ad-40cb-ae54-a49ca03adfc5",
-      "Name": "Encoding",
-      "Type": "String",
-      "Label": "",
-      "HelpText": null,
-      "DefaultValue": "UTF8",
+      "Id": "114781a9-2d44-4841-8bda-2a81a057b420",
+      "Name": "Content",
+      "Label": "Content you wish to write on newly created file.",
+      "HelpText": "Content you wish to write on newly created file.",
+      "DefaultValue": "",
       "DisplaySettings": {
-        "Octopus.ControlType": "Select",
-        "Octopus.SelectOptions": "UTF8|UTF-8\nASCII|ASCII (7-bit)\nBigEndianUnicode|UTF-16 (big-endian)\nByte|Encodes as byte sequence\nUnicode|UTF-16 (little-endian)\nUTF7|UTF-7\nUnknown|Binary"
+        "Octopus.ControlType": "SingleLineText"
```

Comment:
UX for the file content would probably be better if it was a `MultiLineText` control instead of `SingleLineText`